service/src/main/java/uk/gov/hmcts/reform/fpl/service/orders/generator/C43ChildArrangementOrderDocumentParameterGenerator.java

```diff
@@ -50,8 +50,8 @@ public class C43ChildArrangementOrderDocumentParameterGenerator implements Docmo
         + "necessary.";
 
     public static final String NOTICE_MESSAGE = "Any person with parental responsibility for the child may obtain "
-        + "advice on what can be done to prevent the issue of a passport to the child. They should write to London "
-        + "Passport Office, Globe House, 89 Eccleston Square, LONDON SW1V 1PN.";
+        + "advice on what can be done to prevent the issue of a passport to the child. They should write to "
+        + "Glasgow CPST, HMPO Glasgow, 96 Milton Street, Glasgow, G4 0BT or email Glasgowcaveats@hmpo.gov.uk.";
```

Comment:
would it be possible to move the address/email to a config property/env var, so that it's easier to replace if it changes again

Reply:
moved the address/email to application.yaml